data/mayurstwt.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Mayur Sankhala",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Happy since learned development",
+  "avatar": "https://github.com/mayurstwt.png"
```

Comment:
Hey @mayurstwt, our recent update no longer needs the `avatar` key anymore. It will auto-pick your GitHub profile. So please remove this avatar field from your json. 

```suggestion
```